roles/update/tasks/main.yml

```diff
@@ -30,6 +30,14 @@
     - name: Start ping test
       ansible.builtin.include_tasks: l3_agent_connectivity_check_start.yml
 
+- name: Trigger the continuous control plane test
```

Comment:
Overall code looks good. While testing it failed at this step.
```TASK [update : Trigger the continuous control plane test _raw_params={{ cifmw_update_artifacts_basedir }}/control_plane_test_start.sh
2024-09-02 03:52:02.326098 | controller | ] ***
2024-09-02 03:52:02.326105 | controller | Monday 02 September 2024  03:52:02 -0400 (0:00:00.028)       0:00:42.120 ******
2024-09-02 03:52:02.326119 | controller | fatal: [localhost]: FAILED! => changed=true
2024-09-02 03:52:02.391941 | controller |   cmd: |-
2024-09-02 03:52:02.392016 | controller |     /home/zuul/ci-framework-data/tests/update/control_plane_test_start.sh
2024-09-02 03:52:02.392025 | controller |   delta: '0:00:00.004461'
2024-09-02 03:52:02.392032 | controller |   end: '2024-09-02 03:52:02.305037'
2024-09-02 03:52:02.392039 | controller |   msg: non-zero return code
2024-09-02 03:52:02.392045 | controller |   rc: 127
2024-09-02 03:52:02.392052 | controller |   start: '2024-09-02 03:52:02.300576'
2024-09-02 03:52:02.392058 | controller |   stderr: '/bin/sh: line 1: /home/zuul/ci-framework-data/tests/update/control_plane_test_start.sh: No such file or directory'
2024-09-02 03:52:02.392071 | controller |   stderr_lines: <omitted>
2024-09-02 03:52:02.392078 | controller |   stdout: ''
2024-09-02 03:52:02.392084 | controller |   stdout_lines: <omitted>
```
`control_plane_test_start.sh` is created when `Create the support files for test` is called. But `Create the support files for test` will be called only when `cifmw_update_ping_test` is true.

I think we need to include `cifmw_update_control_plane_check | bool` at [Create the support files for test](https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/update/tasks/main.yml#L17) task so that required script get created.

Reply:
Nice catch.  I've updated the code to make sure all the cases were taken into account properly.